src/codemirrorCommands.ts

```diff
@@ -97,198 +123,235 @@ export class VimCellManager {
         }: IKeybinding) => {
           if (keybindEnabled) {
             if (mapfn === 'map') {
-              lvim.map(command, keys, context);
+              Vim.map(command, keys, context);
             } else {
-              lvim.noremap(command, keys, context);
+              Vim.noremap(command, keys, context);
             }
           }
         }
       );
 
       Vim.handleKey(lcm, '<Esc>');
```

Comment:
It really seems as though this line should be putting us into normal mode. but it seems not to be